contrib/actions/cds-split-download.yml

```diff
@@ -0,0 +1,51 @@
+version:            v1.0
+name:               CDS_SplitDownloadArtifact
+description:        "Download Artifacts which were uploaded using CDS_SplitUploadArtifact action."
+parameters:
+  destinationFile:
+    type:           string
+    description:    Destination File to save the file. (Mandatory). (like dist.tar.gz)
+  pattern:
+    type:           string
+    description:    Prefix pattern to identify files to be downloaded. (Mandatory). (like bigfile-*)
+  prefixHandle:
+    type:           string
+    description:    Prefix of the Artifacts uploaded. (Mandatory).
+  tag:
+    type:           string
+    description:    Tag to identify uploaded artifacts. Default tag value is CDS run version number. (Optional)
+requirements:
+  - binary:         bash
+steps:
+- script:
+  - '#!/bin/bash'
+  - set -e
+  - ""
+  - DESTINATION="{{.destinationFile}}"
+  - PATTERN="{{.pattern}}"
+  - PREFIX_HANDLE="{{.prefixHandle}}"
+  - TAG="{{.cds.version}}"
+  - ""
+  - if [[ -z "${DESTINATION}" || -z "${PATTERN}" || -z "${PREFIX_HANDLE}" ]]; then
+  - "\techo -e \"\nTerminating Split-Download Artifact\""
+  - "\techo -e \"Please check if any action parameters are missing...\n\""
+  - "\texit 1"
+  - else
+  - "\techo -e \"\nInitiating Split-Download Artifact on: ${DESTINATION}...\n\""
+  - fi;
+  - ""
+  - if [[ ! -z "{{.tag}}" ]]; then
+  -   'TAG="{{.tag}}"'
+  - fi;
+  - ""
+  - 'worker download --tag=${TAG} --pattern="${PATTERN}"'
+  - ""
+  - 'cat ${PREFIX_HANDLE}-* > ${DESTINATION}'
+  - 'y=$(md5sum ${DESTINATION} | cut -d " " -f 1)'
+  - ""
+  - if grep -Fxq "${y}" *-md5 ; then
+  - "\techo -e \"\nDownloaded Artifacts Are Matching...\""
+  - else
+  - "\techo -e \"\nDownloaded Artifacts Are Not Matching...\n\""
```

Comment:
no exit 1 ?

Reply:
@blaryjp Thanks for pointing that out. I'll add that.